axonserver/src/main/java/io/axoniq/axonserver/component/processor/ClientEventProcessorInfo.java

```diff
@@ -29,6 +30,7 @@ public ClientEventProcessorInfo(String clientId, String clientStreamId, String c
         this.clientStreamId = clientStreamId;
         this.context = context;
         this.eventProcessorInfo = eventProcessorInfo;
+        this.timestamp = System.currentTimeMillis();
```

Comment:
It'd be nice that this class remains the simple data holder :)

Reply:
removed timestamp from here.